stdlib/experimental/experimental.flux

```diff
@@ -1252,3 +1252,33 @@ builtin histogram : (
         bins: [float],
         ?normalize: bool,
     ) => stream[{T with _value: float, le: float}]
+
+// preview limits the number of rows and tables in the stream.
+//
+// Included group keys are not deterministic and depends on the order
+// that the engine sends them.
+//
+// ## Parameters
+// - nrows: Maximum number of rows per table to return. Default is `5`.
+//
+// - ntables: Maximum number of tables to return.
+//   Default is `5`.
+//
+// - tables: Input data. Default is piped-forward data (`<-`).
+//
+// ## Examples
+//
+// ### Preview data output
+// ```
+// import "experimental"
+// import "sampledata"
+//
+// sampledata.init()
+//     |> experimental.preview()
```

Comment:
- Change `sampledata.init()` to `sampledata.int()`
- Passing parameters in the example will show exactly how the input tables are modified.
- Add input (`<`) and output (`>`) identifiers to generate the input and output examples.

```suggestion
// < sampledata.int()
// >     |> experimental.preview(nrows: 3, ntables: 1)
```